cli/cmd/upgrade.go

```diff
@@ -16,6 +16,9 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const okMessage = "You're on your way to upgrading Linkerd!\nVisit this URL for further instructions: https://linkerd.io/upgrade/#nextsteps"
+const failMessage = "For troubleshooting help, visit: https://linkerd.io/upgrade/#troubleshooting"
```

Comment:
I think we can just combine these two lines into a `const` block:
```
const (
  okMessage = "You're on your way to upgrading Linkerd!\nVisit this URL for further instructions: https://linkerd.io/upgrade/#nextsteps"
  failMessage = "For troubleshooting help, visit: https://linkerd.io/upgrade/#troubleshooting"
)
```

Reply:
thanks @ihcsim!